ui/src/app/cohort-review/cohort-review/cohort-review.component.ts

```diff
@@ -24,6 +24,7 @@ export class CohortReviewComponent implements OnInit, OnDestroy {
   ) {}
 
   ngOnInit() {
+    this._broadcast();
```

Comment:
I like the underscore because its familiar from Python, tbh, and because I don't need to look up the declaration site of the property to determine if its private or public.  That being said, I'm good with keeping or ditching, but I agree that we have a lot of inconsistency on this point in the codebase - I think basically that either every and all private attributes / methods should be underscore prefixed or none, but either scenario is ok.

Reply:
Ok.  I'm going to open a separate PR to handle all my `_` prefixes if thats ok.  It would add a lot of noise to this already pretty bulky PR.